server/komodo-rest/src/main/java/org/komodo/rest/service/KomodoDataserviceService.java

```diff
@@ -217,7 +217,7 @@ public Response getDataservices( final @Context HttpHeaders headers,
                 throw ( KomodoRestException )e;
             }
 
-            return createErrorResponse(mediaTypes, e, DATASERVICE_SERVICE_GET_DATASERVICES_ERROR);
+            return createErrorResponse(Status.FORBIDDEN, mediaTypes, e, DATASERVICE_SERVICE_GET_DATASERVICES_ERROR);
         }
```

Comment:
Right I think there were just a few places using it.  The new KomodoService method made it unnecessary, with the user explicitly passing Status.FORBIDDEN (or some other Status).